lib/views/home_content/feed/post_card/post_card.dart

```diff
@@ -30,7 +33,10 @@ class PostCard extends StatelessWidget {
       child: Row(
```

Comment:
Here I think we could just order the local variables as they would appear in the view, so placing the `postBottomBar` at the end of the build would facilitate code reading.

Reply:
Thanks for the advice !
Fixed in e6c59a96bc835188c350722a647fef6e163bee69.